Appraisals

```diff
@@ -62,7 +62,7 @@ elsif Gem::Version.new('1.9.3') <= Gem::Version.new(RUBY_VERSION) \
       gem 'activerecord-mysql-adapter', platform: :ruby
       gem 'aws-sdk', '~> 2.0'
       gem 'dalli'
-      gem 'elasticsearch-transport'
+      gem 'elasticsearch'
```

Comment:
Why did we change this dependency?

Reply:
`elasticsearch` gem contains `elasticsearch-transport` https://github.com/elastic/elasticsearch-ruby/blob/master/elasticsearch/elasticsearch.gemspec#L25

And it provides much higher level API allowing easy use of ES documentation examples as bases for tests.

e.g.
```subject(:request) { client.cluster.health }```
or 
```subject(:request) { client.index index: index_name, type: document_type, id: document_id, body: document_body }```

Before we were using lower level api like:
```client.perform_request 'GET', '_cluster/health'```

Reply:
I don't think we want to broaden our support to `elasticsearch` as a whole unless there's a strong reason for doing so. Is there a reason why we need to broaden this to support the change you're making? Is there not an analogous way to do so with just `elasticsearch-transport`? If so, how is that the case?

Reply:
I've been wary of using low lever API of `elasticsearch-transport` to avoid making the tests understand too much of ES internals. However since we are already tagging relatively low level things by using ES url etc, so we're already quite deep into ES internals.

I've switched the test to only `elasticsearch-transport` to as you pointed out avoid broadening of the support to another gem.